### DISCUSSION TITLE
45d Floor pieces

### DISCUSSION THREAD

**Discussion by chrisanderso** [BentoGambin/Valheim-MissingPieces#2] [Ideas]
first of iv been waiting for this for ages good work so far. we need the thatch roof parts that match the ones you have in the tar series . id love a inside stair cast corner kind of the opposite from what you already have . floor tiles that are cut in half so you have a 45 degree triangle .

chris anderson

**Comment by BentoGambin** [comment]
The 45 degree floors are on the way! Can you elaborate on what you mean by 'inside stair cast corner'? And the roofs are 90% made, I'm just waiting on a bug in piece manager to be solved.

**Reply by chrisanderso** [reply]
no problem so the corner staircase you have already made fits on the out side imagen if you were wanting to make a rectangular shape with stairs leading down towards the center . this kind of thing can be done with clipping the original parts but it is messy . and not very easy to put in.
